libs/sdk-backend-tiger/src/convertors/fromBackend/CatalogConverter.ts

```diff
@@ -71,12 +72,16 @@ export const convertAttribute = (
 };
 
 export const convertMeasure = (measure: JsonApiMetricOutWithLinks): ICatalogMeasure => {
-    const maql = measure.attributes?.content?.maql;
+    const { maql, format = "" } = measure.attributes?.content;
```

Comment:
What if `content` is undefined, will the destructuring fail?

Reply:
@no23reason fixed